tests/unit/info.tcl

```diff
@@ -281,18 +281,12 @@ start_server {tags {"info" "external:skip"}} {
                     r eval {return redis.error_reply(string.format('%s my error message', ARGV[1]))} 0 $j
```

Comment:
Can you cover both the API case and the non-API case?

```
./valkey-cli eval "return {err = 'GREEN My very special reply error'};" 0
./valkey-cli eval "return redis.error_reply('GREEN My very special reply error');" 0
```

Reply:
Sure. Will add test cases for these

Reply:
https://github.com/valkey-io/valkey/pull/500/commits/1c77f69b16f675cf15a5ce5d038d07a831862dfb